app/assets/javascripts/admin/addon/components/color-input.js

```diff
@@ -22,17 +22,31 @@ export default Component.extend({
     return this.onlyHex ? 6 : null;
   }),
 
+  normalize(color) {
+    if (/^#?([0-9a-fA-F]{3}|[0-9a-fA-F]{6})$/.test(color)) {
```

Comment:
I _think_ you can reduce the regexp to this

```suggestion
    if (/^#?([0-9a-f]{3}){1,2}$/i.test(color)) {
```